lib/spack/spack/cmd/spec.py

```diff
@@ -29,7 +29,13 @@
 import spack.cmd
 import spack.cmd.common.arguments as arguments
 
-description = "show what would be installed, given a spec"
+description = """show what would be installed, given a spec"""
+
+help = """
```

Comment:
I don't understand how this works. Shouldn't this variable be called `longdescription`?